From cf6e0d16ac0c0d521da98f5d9a53532b923a621c Mon Sep 17 00:00:00 2001
Message-Id: <cf6e0d16ac0c0d521da98f5d9a53532b923a621c.1549521103.git.liu.denton@gmail.com>
In-Reply-To: <cover.1549521103.git.liu.denton@gmail.com>
References: <cover.1549450636.git.liu.denton@gmail.com>
	<cover.1549521103.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Sat, 12 Jan 2019 13:30:44 -0500
Subject: [PATCH v2 1/3] git-submodule.txt: document default behavior without
 --branch
To: git@vger.kernel.org
Cc: gitster@pobox.com

This behavior is mentioned in gitmodules.txt but not in
git-submodule.txt so we copy the information over so that it is not
missed.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 Documentation/git-submodule.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/git-submodule.txt b/Documentation/git-submodule.txt
index ba3c4df550..9951c68744 100644
--- a/Documentation/git-submodule.txt
+++ b/Documentation/git-submodule.txt
@@ -261,7 +261,8 @@ OPTIONS
 	The name of the branch is recorded as `submodule.<name>.branch` in
 	`.gitmodules` for `update --remote`.  A special value of `.` is used to
 	indicate that the name of the branch in the submodule should be the
-	same name as the current branch in the current repository.
+	same name as the current branch in the current repository.  If the
+	option is not specified, it defaults to 'master'.
 
 -f::
 --force::
-- 
2.20.1.522.g5f42c252e9

